packages/wine.rb

```diff
@@ -108,9 +111,9 @@ def self.remove
       case $stdin.getc
       when 'y', 'Y'
         FileUtils.rm_rf config_dir
-        puts "#{config_dir} removed.".lightred
+        puts "#{config_dir} removed.".lightgreen
       else
-        puts "#{config_dir} saved.".lightgreen
+        puts "#{config_dir} saved.".lightblue
       end
     end
   end
```

Comment:
Add `def self.check` section?

Reply:
I'd love to, when we get it to built first

Reply:
done.